app/views/devise/registrations/_data_protection_policy.html.md

```diff
@@ -0,0 +1 @@
+**TODO:** Fill this with an actual data protection policy
```

Comment:
Ah, yes. This can be deleted, as I removed the reference in https://github.com/theodi/member-directory/commit/04c7e6bdfcd68d2c644476645bcbc9196924e9f9